sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -163,3 +163,21 @@ export class ApplicationSummaryAPIOutDTO {
   submitted?: Date;
   status: ApplicationStatus;
 }
+
+/**
+ * History of SIN validations associated with a user.
+ */
+export class SINValidationsAPIOutDTO {
+  id: number;
+  sin: string;
+  createdAt: Date;
+  isValidSIN?: boolean;
+  sinStatus?: string;
+  validSINCheck?: string;
+  validBirthdateCheck?: string;
+  validFirstNameCheck?: string;
+  validLastNameCheck?: string;
+  validGenderCheck?: string;
+  temporarySIN: boolean;
+  sinExpireDate?: string;
```

Comment:
Please change to sinExpiryDate?

Reply:
The problem started in the DB migration. The field is wrongly named on DB as `sin_expire_date`.
I will change the migration and execute the command directly on DEV to rename it and make it right. @dheepak-aot, any problem if I do it?

Reply:
Renamed all the occurrences for sinExpiryDate,